index.d.ts

```diff
@@ -869,13 +869,48 @@ declare namespace tracer {
     flush(): void
   }
 
+  export interface EventTrackingV2 {
+    /**
+     * Links a successful login event to the current trace. Will link the passed user to the current trace with Appsec.setUser() internally.
+     * @param {string} login the login data (username, email...) used by the user to authenticate
+     * @param {User} user Properties of the authenticated user. Accepts custom fields. Can be null.
+     * @param {[key: string]: string} metadata Custom fields to link to the login success event.
+     */
+    trackUserLoginSuccess(login: string, user?: User | null, metadata?: any): void;
```

Comment:
why any ?
```suggestion
    trackUserLoginSuccess(login: string, user?: User | null, metadata?: { [key: string]: string }): void;
```

Reply:
it really accepts anything, non string values are converted to strings and deeper object are also accepted.